### PR TITLE
Ports: Fix build_installed.sh script

### PR DESCRIPTION
**Pull request by circl-lastname**
And by yak-stension make .hosted_defs.sh usable outside of .port_includes.sh

### CODE REVIEW

File: Ports/build_installed.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
+source .hosted_defs.sh
```
Comment (EWouters) [inline Ports/build_installed.sh:2]: I think we use the `.` syntax to source scripts.

Comment (circl-lastname) [reply Ports/build_installed.sh:2]: Alright

**Comment by EWouters** [comment]
LGTM

**Review by GMTA — APPROVED** [review]
LGTM!

**Pull request merged**